libshaderc/include/shaderc.h

```diff
@@ -105,6 +113,12 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set the target shader environment, affecting which warnings or errors will be issued.
+// The version is currently unused, but will be for distinguishing between different
```

Comment:
If we say version is unused, then people will put random values there.  Better to say only 0 is supported for now.